### PR TITLE
Feature: Search Commpks by commpkNo

### PR DESCRIPTION
**Pull request by herda1**
- Added lookupGroupByCommPkNos

### CODE REVIEW

File: src/packageVersion.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
 /* eslint-disable */
 // auto-generated from rollup.config.js, will be overwritten next production build. Version taken from package.json
-export const packageVersion = '0.1.32';
\ No newline at end of file
+export const packageVersion = '0.1.321';
```
Comment (OveBDev) [inline src/packageVersion.ts:3]: If you want to release this version, call it 1.33. 
3 digits in last number if you are not gonna release this one

**Review by OveBDev — APPROVED** [review]

File: src/offlineSync/commPacksSyncer/commPacksRepository.ts [hunk 2]
```diff
@@ -34,3 +38,10 @@ export function commPacksAdministrator(): DatabaseAdministrator<CommPackDb> {
 export function commPacksRepository(): Repository<CommPackDb> {
     return commPacksAdministrator().repositoryTransaction();
 }
+
+export async function getLocalCommPacksGroupedByCommPkNo(commPackNos: string[]): Promise<Dictionary<CommPackDto[]>> {
+    if (!_tableInstance) return {};
+    return chain(await _tableInstance.where('commPkgNo').anyOf(commPackNos).toArray())
```
Comment (OveBDev) [inline src/offlineSync/commPacksSyncer/commPacksRepository.ts:44]: we might want to get this from inMemory instead in the future. But we don't need to do that now. We have to measure performance first to see if it's needed

**Review by OveBDev — APPROVED** [review]

File: src/offlineSync/commPacksSyncer/commPacksRepository.ts (hunk 2 above)
Comment (herda1) [reply src/offlineSync/commPacksSyncer/commPacksRepository.ts:44]: Where can I get this instead? This function is used to show the statuses in the legend

Comment (OveBDev) [reply src/offlineSync/commPacksSyncer/commPacksRepository.ts:44]: We have to code it in inMemory. So we can just get it from dexie like you have done here for now

**Pull request merged**